src/content/posts/multiplayer_online_game.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: "Trying to make an online multiplayer minigame"
 summary: "A simple article to explain how we made a multiplayer online game using python and what we learnt while doing it, from the very basic use of sockets, to the different communication protocols and a bit of optimization."
-date: 2024-03-11T16:39:09-02:00
```

Comment:
Why modify this? Was this a mistake?

Reply:
The date that was written was not obtained using the date command previously, so the format was not exactly the same. I thought it was better to change it.

Reply:
that's nice!